### PR TITLE
Implement action

### PR DESCRIPTION
**Pull request by bacherfl**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,90 @@
...
+| GKEClusterName | yes | string | Name of the GKE Cluster | - |
+| GKEClusterMachineType | no | string | Name of the machine type used for the GKE Cluster | `n1-standard-4` |
+| GKEClusterNumNodes | no | int | Number of nodes for the GKE Cluster | `3` |
+| GCloudComputeZone | no | string | Name of the GCloud compute Zone | `us-east1-b` |
+| GCloudServiceKey | yes | string | Base64 encoded GCloud service Key JSON | - |
+
+
+### Outputs
+| Name | Format | Description |
+|--|--|--|--|
```
Comment (warber) [inline README.md]: fix formatting of md table

File: action.yml [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+        KEPTN_ENDPOINT=http://$(kubectl -n keptn get service api-gateway-nginx -o jsonpath='{.status.loadBalancer.ingress[0].ip}')/api
+
+        KEPTN_API_TOKEN=$(kubectl get secret keptn-api-token -n keptn -ojsonpath={.data.keptn-api-token} | base64 --decode)
+
+        echo "KEPTN_ENDPOINT=${KEPTN_ENDPOINT}"
+
+        auth_at_keptn $KEPTN_ENDPOINT $KEPTN_API_TOKEN
+        verify_test_step $? "Could not authenticate at Keptn API"
+
+        echo "##[set-output name=KEPTN_ENDPOINT;]$(echo ${KEPTN_ENDPOINT})"
```
Comment (warber) [inline action.yml:122]: Does this mean that these two lines are visible in the log output?

Comment (bacherfl) [reply action.yml:122]: no, they are not

File: README.md (hunk 1 above)
Comment (bacherfl) [reply README.md]: fixed

**Review by warber — APPROVED** [review]

**Pull request merged**